ledger/cow.go

```diff
@@ -65,12 +66,20 @@ type stateDelta struct {
 	hdr *bookkeeping.BlockHeader
 }
 
+// Txids returns the map fo the transactions Ids included
+func (st *StateDelta) Txids() map[transactions.Txid]struct{} {
```

Comment:
Does it make sense just to expose `txids`? Doesn't seem like a getter is doing anything very meaningful here...
Alternatively, if you want to make the `txids` read-only, then perhaps it makes more sense to instead expose a `isDup` method that lets you check whether a txID is itself a duplicate.

Reply:
I'd like to keep it as simple as possible. Adding isDup() would prevent us from iterating on the map. I don't mind exposing the map variable directly either.